test/boulder-tools/setup-govulncheck.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -feuxo pipefail
+
```

Comment:
Consider adding an `EXIT` trap like we have in our other shell scripts:

```bash
STATUS="FAILURE"

function print_outcome() {
  if [ "$STATUS" == SUCCESS ]
  then
    echo -e "\e[32m"$STATUS"\e[0m"
  else
    echo -e "\e[31m"$STATUS"\e[0m while running \e[31m"$STAGE"\e[0m"
  fi
}

# On EXIT, trap and print outcome
trap "print_outcome" EXIT

(do some work)

# Because set -e stops execution in the instance of a command or pipeline
# error; if we got here we assume success
STATUS="SUCCESS"
```

Reply:
I've added a very basic exit trap.